------------------------------------------------------------
revno: 14122
revision-id: squidadm@squid-cache.org-20161209061551-361ava4lrrmbwiy9
parent: squid3@treenet.co.nz-20161209043304-krtzvsm4a0zbzgi8
committer: Source Maintenance <squidadm@squid-cache.org>
branch nick: 3.5
timestamp: Fri 2016-12-09 06:15:51 +0000
message:
  SourceFormat Enforcement
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squidadm@squid-cache.org-20161209061551-\
#   361ava4lrrmbwiy9
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: cb4bfe0e0aaf3e3d107ffb16e2729c6f46d5a822
# timestamp: 2016-12-09 06:51:04 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20161209043304-\
#   krtzvsm4a0zbzgi8
# 
# Begin patch
=== modified file 'src/servers/HttpServer.cc'
--- src/servers/HttpServer.cc	2016-12-09 01:58:33 +0000
+++ src/servers/HttpServer.cc	2016-12-09 06:15:51 +0000
@@ -170,7 +170,7 @@
 bool
 Http::Server::writeControlMsgAndCall(ClientSocketContext *context, HttpReply *rep, AsyncCall::Pointer &call)
 {
-    // Ignore this late control message if we have started sending a 
+    // Ignore this late control message if we have started sending a
     // reply to the user already (e.g., after an error).
     if (context->reply) {
         debugs(11, 2, "drop 1xx made late by " << context->reply);

